private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -1381,6 +1381,13 @@ export class TemplateServiceClient {
     return router;
   }
 
+  public configExpressMiddleware(): Router {
+    var router = express.Router();
+    router.get("/", (_req: Request, res: Response, _next: NextFunction) => {
+      return res.status(200).json({ redirectUri: process.env.ACMS_REDIRECT_URI, appId: process.env.ACMS_APP_ID });
```

Comment:
Can this function return the App Insights Instrumentation key as well please?

Reply:
Yep!